examples/llm/benchmarks/disagg_multinode.yaml

```diff
@@ -31,39 +25,41 @@ Processor:
   # 3. kv: finding prefill workers by KV cache is not beneficial when caching is
   #        disabled on this setup
   router: round-robin
+  # Number of tokens in a batch for more efficient chunked transfers to GPUs.
+  block-size: 128
+  max-model-len: 3500
+  max-num-batched-tokens: 3500
+  gpu-memory-utilization: 0.95
+  disable-log-requests: true
+
+Frontend:
+  served_model_name: neuralmagic/DeepSeek-R1-Distill-Llama-70B-FP8-dynamic
+  endpoint: dynamo.Processor.chat/completions
+  port: 8000
+
+Processor:
+  common-configs: [model, block-size, max-model-len, router]
 
 Router:
-  model-name: neuralmagic/DeepSeek-R1-Distill-Llama-70B-FP8-dynamic
+  common-configs: [model]
   min-workers: 1
 
 VllmWorker:
-  model: neuralmagic/DeepSeek-R1-Distill-Llama-70B-FP8-dynamic
-  kv-transfer-config: '{"kv_connector":"DynamoNixlConnector"}'
-  block-size: 128
-  max-model-len: 3500
+  common-configs: [model, kv-transfer-config, router, block-size, max-model-len, gpu-memory-utilization, disable-log-requests]
```

Comment:
question: I'm not sure that gpu memory utilization would be shared between the prefill and decode worker ... - if we use prefill for context only and no kv cache - does this still take up memory - or is not used?

Should we separate out for clarity? Do they need to match?

Reply:
I don't think the memories are shared between workers. It will still take up memory, by setting a larger gpu memory utilization, but it may not be used.

They don't need to match, and we can separate out and comment for clarity.